index.html

```diff
@@ -4,106 +4,374 @@
+          <svg
+            data-js="bookmark"
+            class="card-set__bookmarklabel"
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            viewbox="0 0 24 24"
+          >
+            <path d="M18 24l-6-5.269-6 5.269v-24h12v24z" />
+          </svg>
+
+          <h2 class="bold">Question</h2>
+          <p class="card-set__blocksatz">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua?
+          </p>
+
+          <button data-js="btn-answer" class="button" type="button">
+            <span data-js="showAnswer">show</span>
+            <span data-js="hideAnswer" class="hidden">hide</span>
+            answer
+          </button>
+          <h2 data-js="Answer-heading" class="bold hidden">Answer</h2>
+          <p data-js="answer-text" class="card-set__blocksatz hidden">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et.
+          </p>
+
+          <ul class="card-set__ul-card">
+            <li class="card-set__tag">#html</li>
+            <li class="card-set__tag">#css</li>
+            <li class="card-set__tag">#grid</li>
+            <li class="card-set__tag">#responsive</li>
+          </ul>
+        </div>
+        <div data-js="card-set" class="card-set">
+          <svg
+            data-js="bookmark"
+            class="card-set__bookmarklabel"
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            viewbox="0 0 24 24"
+          >
+            <path d="M18 24l-6-5.269-6 5.269v-24h12v24z" />
+          </svg>
+
+          <h2 class="bold">Question</h2>
+          <p class="card-set__blocksatz">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua?
+          </p>
+
+          <button data-js="btn-answer" class="button" type="button">
+            <span data-js="showAnswer">show</span>
+            <span data-js="hideAnswer" class="hidden">hide</span>
+            answer
+          </button>
+          <h2 data-js="Answer-heading" class="bold hidden">Answer</h2>
+          <p data-js="answer-text" class="card-set__blocksatz hidden">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et.
+          </p>
+
+          <ul class="card-set__ul-card">
+            <li class="card-set__tag">#html</li>
+            <li class="card-set__tag">#css</li>
+            <li class="card-set__tag">#grid</li>
+            <li class="card-set__tag">#responsive</li>
+          </ul>
+        </div>
+      </div>
+    </main>
+
+    <main data-js="main-create" class="main">
```

Comment:
I belive you dont need a extra class to style that main. If that fits better in your workflow, that should be absolutely fine, its was just something ive noticed.